docker/images.go

```diff
@@ -39,26 +39,23 @@ func (d *docker) ImageRemove(ctx context.Context, imageID string, force, pruneCh
 }
 
 func (d *docker) getImageStatus(ctx context.Context, image types.ImageSummary) (status string) {
-	if len(image.RepoTags) == 0 {
+	if image.RepoTags[0] == "<none>:<none>" {
```

Comment:
There are cases that there might be no RepoTags, in that case, if you want to access the first element in the slice, we'll face a panic.
<img width="469" alt="image" src="https://github.com/arshamalh/dockeroller/assets/81646632/682e26a4-5a37-48d9-b263-1334b46243e0">

Reply:
There is a case that all of our image has tag and we don't have any container. then the logic return status with empty string.
what you think about this code:
```go
func (d *docker) getImageStatus(ctx context.Context, image types.ImageSummary) (status string) {
	if len(image.RepoTags) == 0 {
		status = string(entities.ImageStatusUnUsedDangling)
		return
	}

	if image.RepoTags[0] == "<none>:<none>" {
		status = string(entities.ImageStatusUnUsedDangling)
		return
	}
	containers, _ := d.cli.ContainerList(ctx, types.ContainerListOptions{})
	if len(containers) == 0 {
		status = string(entities.ImageStatusUnUsed)
		return
	}
	newImgs := make(map[string][]string)
	for _, cont := range containers {
		if cont.ImageID != image.ID {
			status = string(entities.ImageStatusUnUsed)
		} else {
			newSlice := newImgs[image.ID]
			if newSlice == nil {
				newSlice = make([]string, 0)
			}
			newSlice = append(newSlice, cont.ID)
			newImgs[image.ID] = newSlice

			status = string(entities.ImageStatusInUse)
		}
	}
```